lib/metrics/metrics.ts

```diff
@@ -2,11 +2,19 @@
 // Licensed under the Apache License, Version 2.0. See LICENSE
 
 export const METRICS = {
-  TTFCP: 'firstContentfulPaint',
-  TTFMP: 'firstMeaningfulPaint',
-  TTFCPUIDLE: 'firstCPUIdle',
-  TTI: 'interactive',
-  SI: 'speedIndex',
-  // @todo add in further improvements
-  // VISUALLY_COMPLETE: 'observedLastVisualChange',
+    BT: 'bootup-time',
```

Comment:
all performance metrics. should extent to other measures from pwa, accessibility, best practice and from user timings